glfs.c

```diff
@@ -530,6 +534,14 @@ static int tcmu_glfs_open(struct tcmu_device *dev, bool reopen)
 		goto unref;
 	}
 
+#if GFAPI_VERSION770
+	ret = glfs_fsetxattr(gfsp->gfd, GF_ENFORCE_MANDATORY_LOCK, "set", 8, 0);
```

Comment:
isn't size 8 meant to be the size of the value that you are setting (`"set"` would be 4 with the `'\0'`).

Reply:
Hmm, yeah that's true, I will fix it.
Thanks :-)

Reply:
@nixpanic

BTW, which version should I check here ? It seems the 7dev is not correct.

Thanks.

Reply:
7dev (or now 8dev) is only set on development branch. I would say for now (till 8dev becomes 8.0) it should be 8dev??

Let @nixpanic confirm.

Reply:
Yeah,  the 8.0 seems the correct one.